libretro/nukleargui/app.c

```diff
@@ -123,24 +116,12 @@ int app_event(int poll)
 
 int app_render(int poll)
 {
-    static int prevpoll=0,prevstate=0,reset_state=0;
-    if(prevpoll!=poll || reset_state){nk_clear(ctx);reset_state=0;}
```

Comment:
Do we really want to remove the nk_clear() call that happens after each opening/closing of the keyboard? The nuklear docs say nk_clear should be called. https://rawgit.com/vurtun/nuklear/master/doc/nuklear.html#nuklear/api

Reply:
At the moment it never happens, because app_render is only called with poll 0.

Reply:
Oh right, the reset_state is changed on opening/closing the vkbd. I'll test some more.